doc/connecting-lab.md

```diff
@@ -12,44 +12,72 @@ document explains how to do this.
 ## Token setup
 
 The first step is to generate a token for the lab. This is done by the lab admin,
-and the token is used to submit jobs from pipeline to the lab, and to authenticate
+and the token is used to submit jobs from pipeline to the lab and to authenticate
 LAVA lab callbacks to the pipeline.
+
 Requirements for the token:
-- Description: a string matching the regular expression `[a-zA-Z0-9\-]+`, for example "kernelci-new-api-callback"
-- Value: arbitrary, kept secret
+- `Description`: a string matching the regular expression `[a-zA-Z0-9\-]+`, for example "kernelci-new-api-callback"
+- `Value`: arbitrary, kept secret
 
-*IMPORTANT!* You need to have both fields, as way how LAVA works:
-- You submit the job with the token-description in job definition
-- LAVA lab sends the result back to the pipeline with the token-value (retrieved by that token-description) in the header
+*IMPORTANT!* You need to have both fields, as that's how LAVA works:
+- You submit the job with the token description in job definition
+- LAVA lab sends the result back to the pipeline with the token value (retrieved by that token-description) in the header
 
 More details in [LAVA documentation](https://docs.lavasoftware.org/lava/user-notifications.html#notification-callbacks).
 
 
 ## Pipeline configuration
 
-### Secrets (toml) file
+### Update pipeline configuration
+
+The first step is to add the lab configuration to [pipeline configuration](https://github.com/kernelci/kernelci-pipeline/blob/main/config/pipeline.yaml) file.
+
+Please add a new entry to the `runtimes` section of the configuration file
```

Comment:
I understand why there's no direct link to the runtime section - its position within the file might alter in time.

If the release model for this setup moves to git tags, it could be useful to provide a direct link to the tagged version of the configuration file.

Reply:
I am not sure if we do that as what if we decide to remove a tagged version later on?